cves/CVE-2015-1831.yml

```diff
@@ -173,11 +176,11 @@ lessons:
     applies:
     note:
   native_wrappers:
-    applies:
-    note:
+    applies: true
+    note: It was because of a function that overrode the DefaultExcludedPatternsChecker that made the the hole appear.
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: This arose because they were trying to create their own function to sanitize input coming in, but ended up opening a hole when implementing it that attackers could use.
```

Comment:
Consider rewording this to sound more professional, such as replacing "they" with "the developers" and "opening a hole" with "creating a vulnerability"

Reply:
The description has been edited to sound better